MoltenVK/MoltenVK/Commands/MVKCommandResourceFactory.mm

```diff
@@ -286,7 +286,7 @@ static void getSwizzleString(char swizzleStr[4], VkComponentMapping vkMapping) {
 		[msl appendLineMVK: @"                         constant TexSubrez& subRez [[buffer(0)]]) {"];
 		[msl appendLineMVK: @"    FragmentOutputs out;"];
 		if (mvkIsAnyFlagEnabled(blitKey.srcAspect, (VK_IMAGE_ASPECT_DEPTH_BIT))) {
-			[msl appendFormat: @"    out.depth = tex.sample(ce_sampler, varyings.v_texCoord%@%@, level(subRez.lod)).%c;", coordArg, sliceArg, swizzleArg[0]];
+			[msl appendFormat: @"    out.depth = tex.sample(ce_sampler, varyings.v_texCoord%@%@, level(subRez.lod));", coordArg, sliceArg];
 			[msl appendLineMVK];
 		}
 		if (mvkIsAnyFlagEnabled(blitKey.srcAspect, (VK_IMAGE_ASPECT_STENCIL_BIT))) {
```

Comment:
Stencil blits are broken, too.

Reply:
>Stencil blits are broken, too.

Not in CTS. I deliberately looked for that. All BLIT tests pass, including stencil versions. 

Plus, the reports from tests that failed that included both depth & stencil, were all in the form:

```
    out.depth = tex.sample(ce_sampler, varyings.v_texCoord.xy, level(subRez.lod)).x;
    out.stencil = stencilTex.sample(ce_stencil_sampler, varyings.v_texCoord.xy, level(subRez.lod)).x;
    return out;
}

program_source:26:82: error: member reference base type 'float' is not a structure or union
    out.depth = tex.sample(ce_sampler, varyings.v_texCoord.xy, level(subRez.lod)).x;
                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~
```

with no mention of stencil issues. And these tests passed once the depth was fixed.

It appears that stencil samples are treated like an `R8` format, with a `uchar4` result.

Reply:
Huh. Never mind, then.